ghcide.cabal

```diff
@@ -90,7 +90,7 @@ library
         ghc-paths,
         cryptohash-sha1 >=0.11.100 && <0.12,
         hie-bios >= 0.7.1 && < 0.8.0,
-        implicit-hie-cradle >= 0.2.0.1 && < 0.3,
+        implicit-hie-cradle >= 0.2.0.1 && < 0.4,
```

Comment:
Maybe we should remove the upper bound? If we are no gonna change the api frequently, maybe it would be handy

Reply:
The API will change if `hie-bios`'s changes, I am more conformable with a upper bound.